intelmq/tests/bots/experts/sieve/test_sieve_files/test_ip_range_list_match.sieve

```diff
@@ -0,0 +1,3 @@
+if source.ip << ['192.0.0.0/24', '169.254.0.0/16'] {
```

Comment:
this does not seem to be documented in the docu. I mean that the ``<<`` operator also works on lists.

Reply:
Actually, every operator works on lists. I will point that out.